packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -96,6 +98,102 @@ const ruleset: any = {
         function: falsy,
       },
     },
+    URIContainsSubscriptionId: {
```

Comment:
I think all these rules should be renamed to "PathContains..." since they check the "path" element.  Yes, the path will be used as a URL in a request, but we should match the terminology with features of the API definition that we are actually checking.

Reply:
I'd like to call it 'URI'  because actually what I want to ensure is the URI not 'path', for example the path may not contains 'subscriptionId' instead it contains '{scope}', that's ok

Reply:
I still think "Path" is the correct term here. Looking at the existing rules, there are none whose name starts with "URI", but several that start with "Path", e.g. "PathResourceProviderMatchNamespace", "PathResourceProviderNamePascalCase", "PathResourceTypeNameCamelCase".  So on precedent alone I think we should be using "Path".

And reiterating my earlier point -- this tool checks an OpenAPI v2 document for compliance to our guidelines. We should use the terminology of OpenAPI v2 to describe problems.  There is not one mention of the term "URI" in the OpenAPI v2 spec. The element in the doc that you flag as in error is a "path". Why not call it a "Path"?

Reply:
@rkmanda Do you have a preference for "URI" vs "Path"? Since these are rules specifically for ARM we should probably let you decide which you think the ARM reviewers will prefer.

Reply:
I have no preference. Good with path.